scripts/no-missing-keys.js

```diff
@@ -0,0 +1,12 @@
+const loginEnBundle = require('../packages/@okta/i18n/src/json/login.json');
+const loginokPLBundle = require('../packages/@okta/i18n/src/json/login_ok_PL.json');
+
+const checkLoginBundleForMissingKeys = () => {
+  const noOfKeysLoginEn = Object.keys(loginEnBundle).length;
+  const noOfKeysLoginokPL = Object.keys(loginokPLBundle).length;
+  if (noOfKeysLoginEn !== noOfKeysLoginokPL) {
```

Comment:
comparing length seems not ideal but run equality check on the list of keys.

Reply:
Yeah, we can update the check to `Object.keys(loginEnBundle) !== Object.keys(loginokPLBundle)`

Reply:
Don't think `===` works for array. Need other utilities. e.g `underscore.isEqual`

<img width="310" alt="Screen Shot 2021-04-12 at 3 37 12 PM" src="https://user-images.githubusercontent.com/13139863/114471141-f73f6180-9ba4-11eb-921b-c14653cdfa41.png">